conf/broker.conf

```diff
@@ -159,6 +159,13 @@ skipBrokerShutdownOnOOM=false
 # Factory class-name to create topic with custom workflow
 topicFactoryClassName=
 
+# Max capacity of the topic name cache.
+topicNameCacheCaxCapacity=100000
+
+# A Specifies the minimum number of minutes that the system stays in the memory, to avoid clear cache frequently when
+# there are too many topics are in use.
+maxSecondsToClearTopicNameCache=7200
```

Comment:
I don't suggest adding new configurations because it requires a PIP for that.

Besides, I don't think there is much need to tune these two configs because we cannot predicate the actual use frequency of topic names.